src/io/flutter/sdk/FlutterSettingsConfigurable.form

```diff
@@ -249,6 +249,15 @@
               <toolTipText value="Provides advanced editing capabilities for Java and Kotlin code. Uses Gradle to find Android libraries then links them into the Flutter project."/>
             </properties>
           </component>
+          <component id="33088" class="javax.swing.JCheckBox" binding="myEnableHotUiCheckBox">
```

Comment:
we could also add a checkbox to enable previews in the code editor but that is more risky as described elsewhere.